pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -54,6 +54,9 @@ const (
 	gceAffinityTypeClientIP = "CLIENT_IP"
 	// AffinityTypeClientIPProto - affinity based on Client IP and port.
 	gceAffinityTypeClientIPProto = "CLIENT_IP_PROTO"
+
+	operationPollInterval        = 3 * time.Second
+	operationPollTimeoutDuration = 7 * time.Minute
```

Comment:
where does the 7 minutes come from?  what's the behavior if 7 minutes is reached and e.g. operation actually succeeds (or fails) after 8 minutes?